web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -373,6 +373,15 @@ export type AwsOidcDeployServiceRequest = {
   accountId: string;
 };
 
+/**
+ * AwsOidcPolicyPreset specifies preset policy to apply
+ * to the AWS IAM role created for the OIDC integration.
+ */
+export enum AwsOidcPolicyPreset {
+  Unspecified = 'unspecified',
```

Comment:
Does it make more sense to use the empty string for the `Unspecified` variant?

```suggestion
  Unspecified = '',
```